### ISSUE TITLE
Unicode instance variable names are not supported

### ISSUE THREAD

**Issue by gogainda** [oracle/truffleruby#2462]
```
% jt ruby -e 'o = Object.new; o.instance_variable_set(:@💙, 42); p o.instance_variable_get(:@💙)'
-e:1:in `instance_variable_set': `@💙' is not allowable as an instance variable name (NameError)
	from -e:1:in `<main>'
```

**Comment by eregon** [comment]
@bjfish and I are working on this, should be fixed soon after the winter holidays

**Comment by eregon** [comment]
@headius https://github.com/test-prof/test-prof/blob/ef07109ecdd43683a3148946eed9389121324442/lib/test_prof/recipes/rspec/let_it_be.rb#L78

**Comment by dblock** [comment]
> @headius https://github.com/test-prof/test-prof/blob/ef07109ecdd43683a3148946eed9389121324442/lib/test_prof/recipes/rspec/let_it_be.rb#L78

Nice!

**Comment by headius** [comment]
There should be no issue with this code on any current version of JRuby.

**Comment by eregon** [comment]
Fixed in d97d1f36b9e657a066633ddba5b8254e58eb47bc by @bjfish and I.

(looks like we forgot a changelog entry, could you add one @bjfish?)
